### ISSUE TITLE
Not Working in Eclipse 2020-12

### ISSUE THREAD

**Issue by scottie-uk** [mihnita/ansi-econsole#60]
I have installed this from the market place on my Eclipse install Eclipse IDE for JavaDevelopers):
Version: 2020-12 (4.18.0)
Build id: 20201210-1552

I see no colored output and no ANSI button on the console indicating that it may not even be running. Any help with debugging or getting this going would be greatly appreciated.

**Comment by mihnita** [comment]
Thank you for the update, and sorry for the delay, I didn't have time until week-end.

Did you have to run it as admin only once, and it fixed the problem?
Or you have to do that all the time? (so it does not work if you don't run as admin)

I am not doing anything in the plugin that requires admin rights, and I'm not writing anything anywhere.
So I suspect an Eclipse thing.

But just in case, can you take a look at the Eclipse error log?
Window --> Show View --> Error Log

I will also try it on Windows "C:/program Files". That is not where I normally put it, but I will try :-)

Thanks for reporting,
M.

**Comment by mihnita** [comment]
Duplicate of #61
